tokens/transformation/skia/index.js

```diff
@@ -0,0 +1,61 @@
+const StyleDictionary = require('style-dictionary')
+const _template = require('lodash/template')
+const fs = require('fs')
+
+StyleDictionary.registerTransform({
+  name: 'color/hex8ToRgbaPartial',
+  ...require('../common/colorToRgbaPartial')
+})
```

Comment:
This shouldn't be necessary since it's already registered by both the web and trailwind transformations. Both of those should probably be eventually refactored as well. We should probably consider importing common transforms explicitly above the other imports in `transformTokens.js`.